packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -110,7 +110,7 @@ export class GroupHeaderBase extends BaseComponent<IGroupHeaderProps, IGroupHead
             type="button"
             className={this._classNames.expand}
             onClick={this._onToggleCollapse}
-            {...expandButtonProps}
+            aria-label={expandCollapseButtonAriaLabel}
```

Comment:
you're taking away the ability for the creator to pass in whatever html button attributes they want. You should not have removed the spread of the expandButtonProps

Reply:
Good catch. I originally didn't see how `expandButtonProps` could be passed in through the top `GroupedList` layer. I'll change that back.